lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -51,7 +51,8 @@
                     header_text: context.header_text,
                     course_id: context.course_id,
                     platform_name: context.platform_name,
-                    account_settings_url: context.account_settings_url
+                    account_settings_url: context.account_settings_url,
+                    hide_username: context.hide_username,
```

Comment:
Complete context is not passed down to the `lms/templates/financial-assistance/financial_assessment_form.underscore`, only the user_details are currently being passed. You need to set the hide_username in the model like its done on line 65.

```
this.model.set(context.user_details);
this.model.set({ hide_username: context.hide_username });
```